docs/core/profiles.md

```diff
@@ -151,96 +153,112 @@ can be influenced by the different fields of such a custom model.
   
 #### Customizing `speed`
 
-For every edge a default speed is inherited from the base vehicle, but you have multiple options to adjust it.
+For every road segment a default speed is inherited from the base vehicle, but you have multiple options to adjust it.
 The first thing you can do is rescaling the default speeds using the `speed_factor` section. For example this is how you
-can reduce the the speed of every edge that has the value 'motorway' for the category 'road_class' to fifty percent of 
+can reduce the speed of every road segment that has the value MOTORWAY for the category 'road_class' to fifty percent of 
 the default speed that is normally used by the base vehicle for this road class:
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5}
-```  
-Note that `road_class: {motorway: 0.5}` is an alternative YAML notation that is equivalent to:
+  road_class == MOTORWAY: 0.5
+```
+
+You can also change the speed factor for multiple road classes like this
 ```yaml
 speed_factor:
-  road_class:
-    motorway: 0.5
+  road_class == MOTORWAY: 0.5
+  road_class == PRIMARY || road_class == TERTIARY: 0.7
 ```
 
-You can also setup speed factors for multiple road classes like this
+Please note the OR operator `||` that says that if one of the expressions is true the value 0.7 is used.
+
+or use multiple categories to influence the speed factor
 ```yaml
 speed_factor:
-  road_class: {motorway: 0.5, primary: 0.7, tertiary: 0.9}
+  road_class == MOTORWAY: 0.5
+  road_environment == TUNNEL: 0.8
 ```
 
-and use multiple categories to influence the speed factor
+You can use only two operators `==` (equality) and `!=` inequality.
```

Comment:
What do you mean here?

Reply:
encoded values cannot other operators like `>` or `<`.

Reply:
Ah ok but I can still use `||` and `&&`? I was a bit confused because this sentence sounds a bit like all I can do is use one condition with `==` or `!=` even though in the example above there is `||` as well.

Reply:
Ah, yes boolean ops are still possible for all boolean operands.

Reply:
And brackets `()` and negation `!` are also fine?

Reply:
Yes :) ... any Java stuff. Should we remove this sentence as it seems to be confusing?

Reply:
Yes, maybe better remove it. To me it would have been clear if the docs said that the keys in the yaml file are boolean conditions written in Java language. For people who do not know Java we could add a short section that briefly explains `==`,`!=`,`&&` and `||` to get them started.